crates/bevy_render/src/camera/projection.rs

```diff
@@ -215,6 +216,45 @@ pub enum ScalingMode {
     FixedHorizontal(f32),
 }
 
+/// Scale the scaling mode. For example, multiplying by 2 makes everything twice as small.
+impl Mul<f32> for ScalingMode {
+    type Output = ScalingMode;
+
+    fn mul(self, rhs: f32) -> ScalingMode {
```

Comment:
I don't think adding the doc-comment to the implementation does anything. So I think it should it should be added to the `mul` method itself, which will allow the user to hover over the `*` and see what the multiplication operator does in this case. I also reworded it a bit in my suggestion.
```suggestion
impl Mul<f32> for ScalingMode {
    type Output = ScalingMode;

    /// Scale the [`ScalingMode`]. Multiplying by `2` makes the viewport twice as large.
    fn mul(self, rhs: f32) -> ScalingMode {
```

Reply:
> I don't think adding the doc-comment to the implementation does anything

It is shown in generated rustdoc:

<img width="681" alt="image" src="https://github.com/bevyengine/bevy/assets/28969/76a76306-f2ba-46cd-afe9-88046ea2b8b4">

Reply:
Anyway, moved the updated doc to `mul`.